helm-chart-sources/pulsar/templates/broker-deployment/broker-configmap.yaml

```diff
@@ -210,6 +210,29 @@ data:
   gcsManagedLedgerOffloadReadBufferSizeInBytes: "{{ .Values.storageOffload.readBufferSizeInBytes }}"
   {{- end }}
 {{- end }}
+{{- if eq .Values.storageOffload.driver "azureblob" }}
+  managedLedgerOffloadDriver: "{{ .Values.storageOffload.driver }}"
```

Comment:
Looks like there is some duplication here between the `google-cloud-storage` and the `azureblob` conditional blocks. We don't need to block this PR, but we should eventually remove the duplication.

Reply:
True, and the same applies to AWS and GCP. It should be improved, I can do that at the next iteration.